addons/crm_livechat/tests/test_crm_lead.py

```diff
@@ -78,8 +79,10 @@ def test_crm_lead_creation_portal(self):
         channel = self.env["discuss.channel"].browse(data["discuss.channel"][0]["id"])
         lead = channel._convert_visitor_to_lead(self.env.user.partner_id, '/lead TestLead command')
 
+        # invalidate_recordset for sudo to avoid cache issues in testing files
+        channel.invalidate_recordset(["channel_member_ids", "channel_partner_ids"])
```

Comment:
Don't you want to also use search here instead?

Reply:
because it is using `channel_partner_ids` but not `channel_member_ids` as others, wanted to make sure the comput for `channel_partner_ids` is working as well. But I can switch to search with `members.partner_id` as well, which is the same as the compute. Just bypassing the compute part from the model.